src/components/dashboard/dashboardv1.test.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { getAllByTestId } from "../../utils/test-utils.js";
+import { BrowserRouter as Router } from "react-router-dom";
```

Comment:
For the testing issues, we just had to import the router for the files.

Reply:
good to know.